Formalisation(shacl)/Core/Example-Data/example-dataset.ttl

```diff
@@ -2,6 +2,16 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix ns1: <http://www.w3.org/2006/vcard/ns#> .
```

Comment:
Non human-friendly namespace prefix.

Reply:
replace ns1 with vcard and deleted ns1 line